README.md

```diff
@@ -1,2 +1,16 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+Display 10 random words    
+Number of attempts = 5     
+X = number of correct letter    
```

Comment:
You were asked to provide pseudocode for the algorithm that calculates the number of correct letters, however you have provided pseudocode for the whole game. Be sure to read the requirements of the worksheet carefully!